recipe/meta.yaml

```diff
@@ -1,35 +1,47 @@
 {% set name = "slycot" %}
-{% set version = "0.3.3" %}
-{% set sha256 = "d98ec859a398af8a3917a0d62e0724996c8b3e780813f8685410525aed9fbafd" %}
+{% set version = "0.3.4" %}
+{% set sha256 = "3ae6db8df6b99b4417394404a548c6464248fc7031f21008f992e317f91d23f1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # for testing still, should switch over to pypi package when available
+  url: https://github.com/python-control/Slycot/archive/v{{ version }}.tar.gz
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 1003
-  # NOTE: Conda forge lacks openblas on Windows, so skip the Windows build.
-  skip: true  # [win]
+  number: 0
```

Comment:
This may need to be 1000, not 0.

Reply:
fine with me, but why does it then say reset the build number to 0?

Reply:
Actually, this is no longer necessary: https://conda-forge.org/docs/maintainer/maintainer_faq.html?highlight=build%20number#mfaq-build-number-1000

The conda forge details change so fast, I can't keep up.

So leave at 0.